api/src/main/java/org/openmrs/module/kenyaemr/chore/UpdateOpenMRSIDAsPreferredIdentifier.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.kenyaemr.chore;
+
+import org.openmrs.api.context.Context;
+import org.openmrs.module.kenyacore.chore.AbstractChore;
+import org.springframework.stereotype.Component;
+
+import java.io.PrintWriter;
+
+/**
+ * update OpenMRS id as preferred identifier
+ */
+@Component("kenyaemr.chore.UpdateOpenMRSIDAsPreferredIdentifier")
+public class UpdateOpenMRSIDAsPreferredIdentifier extends AbstractChore {
+
+    /**
+     * @see AbstractChore#perform(PrintWriter)
+     */
+
+    @Override
+    public void perform(PrintWriter out) {
+        String updateConceptSql = "update patient_identifier pi set pi.preferred = 1 where pi.preferred = 0 and pi.identifier_type = 3;";
```

Comment:
Do you mind joining this to indentifier type table and use the uuid for openmrs ID?

Reply:
Also do this only where preferred is null

Reply:
> Also do this only where preferred is null

The only values are 0 and 1. Edit is failing when preferred is 0

Reply:
> Do you mind joining this to indentifier type table and use the uuid for openmrs ID?
okay.